website/projects/scimaps/src/app/shared/components/book-overview/book-overview.component.scss

```diff
@@ -0,0 +1,40 @@
+.overview-container {
+  display: flex;
```

Comment:
Needs to behave correctly on small screens. @bherr2 have we decided on how to detect small screens? Media queries on width (px, in, cm, etc.)?

Reply:
I think we have to do media queries for breakpoints and something else for mobile vs not-mobile. Any thoughts?

Reply:
I think media queries might work for mobile too, but I'm not sure. Some mobile devices are more like tablets...

Reply:
Maybe we should ask @lecross which layout to use for different sample devices? Selected for the emulated device list in the browser?

Reply:
Since we will have to tweak anyways these are rough estimates. You have the starting place of 1248px for desktop. Break at 960px for tablet. Break at 640px for mobile. The breaks are at roughly 300px increments, which is roughly the equivalent of a column in a four column layout. This should be a good starting place for breakpoints. As stated, we will need to adjust once we see things fully in the wild.

Reply:
I'm going to provide a more granular response sometime today. Bruce is aware.